### DISCUSSION TITLE
Parameter name `no_override` as a boolean is confusing

### DISCUSSION THREAD

**Discussion by chadxz** [bobheadxi/deployments#39] [Ideas] [answered]
In general I recommend keeping your boolean parameter names "positive". In the case of this one, instead of "no_override", name it "override". The current default is true, so you could make the new parameter's default "false".

This requires less cognitive effort to parse what exactly the parameter means. You can even see the confusing bit in the code that uses it:

```ts
// mark existing deployments of this environment as inactive
if (!args.noOverride) {
  await deactivateEnvironment(context, args.environment);
}
```

If you say that out loud, it sounds like "if not no override", which would be simpler written as "if override".

Anyway, just my $0.02. Thanks for this project I'm going to use it on my own repo!

**Comment by bobheadxi** [comment, marked as answer]
This is being renamed to `override` in https://github.com/bobheadxi/deployments/pull/40 !
